code/game/objects/items/devices/instruments.dm

```diff
@@ -302,5 +302,8 @@
 	var/choice = input(M,"What instrument would you like to order?","Jazz Express") as null|anything in display_names
 	if(!choice || !M.canUseTopic(src, BE_CLOSE, FALSE, NO_TK))
 		return
-	var/instrument = new display_names[choice]
+	var/middlevar = display_names[choice]

```

Comment:
...as in name it something indicative of what it's being used for. Or just reuse the choice var instead of defining a new one.

Reply:
choice is good and will remove the silly intermediate var